torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -95,6 +100,17 @@ def validate_args_and_maybe_create_graph_inputs(
                 )
             else:
                 new_arg = a
+        elif isinstance(a, SymNodeVariable):
+            if manually_set_subgraph_inputs:
+                new_proxy = tracer.create_graph_input(str(a.sym_num.node.expr))
+                new_arg = wrap_fx_proxy_cls(
+                    target_cls=SymNodeVariable,
+                    tx=tx,
+                    proxy=new_proxy,
+                    example_value=a.sym_num,
+                )
+            else:
+                new_arg = a
```

Comment:
Both branches are tested:
* ```manually_set_subgraph_inputs=True``` -> ```map```
* ```manually_set_subgraph_inputs=False``` -> ```wrap```

Reply:
Thanks for pointing this out